AdminWebsite/AdminWebsite/Extensions/HearingDetailsResponseExtensions.cs

```diff
@@ -19,6 +19,18 @@ public static bool HasScheduleAmended(this HearingDetailsResponse hearing, Heari
         {
             return hearing.ScheduledDateTime.Ticks != anotherHearing.ScheduledDateTime.Ticks;
         }
+        public static bool HasGenericHearingJudgeChanged(this HearingDetailsResponse hearing,
+            HearingDetailsResponse originalHearing)
+        {
+            var judgeA = hearing.Participants.FirstOrDefault(x =>
```

Comment:
could you change the variable names to e.g: originalHearingJudge, newHearingJudge